common/sgx/eeid.c

```diff
@@ -132,13 +132,12 @@ static oe_result_t deserialize_element(
 oe_result_t oe_serialize_eeid(const oe_eeid_t* eeid, char* buf, size_t buf_size)
 {
     oe_result_t result = OE_UNEXPECTED;
+    char** position = &buf;
```

Comment:
This is an unsafe pointer dereference because the null check happens after this line now.

Reply:
There's no dereference here, it just takes the address of the argument, which should always be safe, even if `buf` is `NULL`.